kubernetes/crd/domain-v1beta1-crd.yaml

```diff
@@ -10643,8 +10654,9 @@ spec:
                 will restart the required servers.
             introspectVersion:
               type: string
-              description: If present, every time this value is updated, the operator
-                will start introspect domain job
+              description: If the domain is running, the operator will restart its
+                introspector job when this value is changed.This field is ignored
```

Comment:
Typo missing space after the period -> ". This field is ignored when the domainHomeSourceType source type is FromModel."

Reply:
This file is generate. It looks correct in Git. Why doesn't it look right in the review?!

Reply:
Could you please check out the branch locally and see if these errors are there?